.github/CODEOWNERS

```diff
@@ -29,13 +29,14 @@
 /c_glib/ @kou
 # /cpp/
 /cpp/src/arrow/acero @westonpace
+/cpp/src/arrow/adapters/orc @wgtmac
 /cpp/src/arrow/dataset @westonpace
 /cpp/src/arrow/engine @westonpace
 /cpp/src/arrow/flight/ @lidavidm
 /cpp/src/arrow/util/async* @westonpace
 /cpp/src/arrow/util/future* @westonpace
 /cpp/src/arrow/util/thread* @westonpace
-/cpp/src/parquet @wjones127
+/cpp/src/parquet @wjones127 @wgtmac
```

Comment:
@wjones127 Do you still want to be listed here?

Reply:
Feel free to directly edit this line. @wjones127 @pitrou